src/zarr/abc/codec.py

```diff
@@ -34,11 +35,15 @@
 CodecOutput = TypeVar("CodecOutput", bound=NDBuffer | Buffer)
 
 
-class _Codec(Metadata, Generic[CodecInput, CodecOutput]):
+class CodecBase(Metadata, Generic[CodecInput, CodecOutput]):
```

Comment:
Feel free to ignore this suggestion, but somehow the name `BaseCodec` seems a bit more natural than `CodecBase`. Maybe it seems a bit more consistent with names like `RemoteStore` which follow the `<Qualification><QualifiedEntity>` schema?

Reply:
👍 I think that's a good suggestion, changed